src/models/index.js

```diff
@@ -0,0 +1,19 @@
+const CatModel = require('./cats');
+
+const Sequelize = require('sequelize');
+
+const setUpDatabase = () => {
+    const connection = new Sequelize("have-i-fed-the-cat-app", "root", "password", {
+        host: "localhost",
+        port: 3307,
```

Comment:
I'm not sure what port you docker container is running on but I suspect it is more likely to be 5432 if you used the docker command from the track. If you connected successfully to the database in PGAdmin then just ignore this comment